applications/nrf_desktop/configuration/nrf52840dongle_nrf52840/app_common.dtsi

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	chosen {
+		/*
+		 * In some default configurations within the nRF Connect SDK,
+		 * e.g. on nRF52840 and nRF9160, the chosen zephyr,entropy node
+		 * is &cryptocell. This devicetree overlay ensures that default
+		 * is overridden wherever it is set, as this application uses
+		 * the RNG node for entropy exclusively.
+		 */
+		zephyr,entropy = &rng;
+	};
+
+	pwmleds {
+		red_pwm_led: pwm_led_0 {
+			pwms = <&pwm0 0 PWM_MSEC(1) PWM_POLARITY_INVERTED>;
+		};
+		green_pwm_led: pwm_led_1 {
+			pwms = <&pwm0 1 PWM_MSEC(1) PWM_POLARITY_INVERTED>;
+		};
+		blue_pwm_led: pwm_led_2 {
+			pwms = <&pwm0 2 PWM_MSEC(1) PWM_POLARITY_INVERTED>;
+		};
+	};
+
+	pwmleds1 {
+		compatible = "pwm-leds";
+		status = "okay";
+
+		pwm_led1: led_pwm_4 {
+			status = "okay";
+			pwms = <&pwm1 0 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
+			label = "LED System State";
+		};
+	};
+};
+
+&usbd {
```

Comment:
btw. This one could be defined per build type (you could align number of USB endpoints to needs of a given configuration). It would also be more consistent with the current approach - USB configuration is defined per build type.

Reply:
Yes, this could be improved in subsequent PRs. I just made sure that the generated zephyr.dts file is exactly the same after this change

Reply:
Improved in the following PR:

https://github.com/nrfconnect/sdk-nrf/pull/19239